config/webpack.config.js

```diff
@@ -146,7 +149,7 @@ const commonConfig = ({ dev }) => {
       // HMR flag
       hot: true,
       ...proxy({
-        env: 'stage-beta',
+        env: 'prod-stable',
```

Comment:
^^

Reply:
I don't have a staging account, do you mind if I keep this here until this is actually ready to go in so that I don't have to change it back every time I want to run this locally?